src/Mammon/Actors/LAWorkspaceActor.cs

```diff
@@ -29,6 +27,10 @@ public async Task SplitCost(string reportId, string resourceId, ResourceCost laT
 				if (item.SelectorType == Consts.ResourceIdLAWorkspaceSelectorType)
 				{
 					//map via resource id
+					///TODO: consider mapping by simply running the selector through the cost centre rule engine (however no tags at this level)
+					///but tags can be retrieved either via resource actor or Azure SDK
+					///this removes the need for large payload to be passed around and simplifies the code as no need to identity/process gaps (previous unseen resource ids) first
+
```

Comment:
is this a user story? :p

Reply:
yes please - but this is likely part of the overall assessment when we do large estate redesign